modules/recommended-etcd-practices.adoc

```diff
@@ -54,3 +54,11 @@ $ sudo docker run --volume /var/lib/etcd:/var/lib/etcd:Z quay.io/openshift-scale
 
 The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile 
 of the fsync metric captured from the run to see if it is less than 10ms.
+
+Etcd replicates the requests among all the members, so its performance strongly depends on network 
+input/output (IO) latency. High network latencies result in etcd heartbeats taking longer than the 
+election timeout, which leads to leader elections that are disruptive to the cluster. Some of the key 
+metrics to monitor on a deployed {product-title} cluster are p99 of etcd network peer latency. Use 
```

Comment:
Is p99 a set of multiple metrics or just one metric? Thinking about how to reword this sentence since it is a little unclear.

Reply:
suggestion:  A key 
metric to monitor on a deployed {product-title} cluster is the 99th percentile of etcd network peer latency on each etcd cluster member.

Reply:
Fixed it, thanks.